src/js/GlobalConstants.js

```diff
@@ -1,16 +1,48 @@
 /**
  * GlobalConstants.js
- * Created by Kevin Li 6/20/17
+ * Created by Maxwell Kendall 7/8/19
  */
 
 // use the correct GlobalConstants file based on the current environment
 
-let isDev = true;
-if (process.env.NODE_ENV && process.env.NODE_ENV === 'production') {
-    isDev = false;
-}
+// Production ENV values
+const prod = {
+    // PROD
+    API: 'https://api.usaspending.gov/api/',
+    LOCAL_ROOT: "",
+    GITHUB: "",
+    GA_TRACKING_ID: "UA-92617810-1",
+    LOCAL: false,
+    DEV: true,
+    KEYWORD_AVAILABLE: true,
+    PERF_LOG: false,
+    OVERRIDE_FISCAL_YEAR: false,
+    FISCAL_YEAR: 2017,
+    MAPBOX_TOKEN:
+    "pk.eyJ1IjoidXNhc3BlbmRpbmciLCJhIjoiY2l6ZnZjcmh0MDBtbDMybWt6NDR4cjR6ZSJ9.zsCqjJgrMDOA-i1RcCvGvg"
+};
 
-const kGlobalConstants = isDev ?
-    require('../../GlobalConstants_dev').default : require('../../GlobalConstants_prod').default;
+// Development ENV values
+const dev = {
+    // LOCAL
+    API: "http://localhost:8000/api/",
+    LOCAL_ROOT: "",
+    GITHUB: "",
+    GA_TRACKING_ID: "UA-92617810-1",
+    LOCAL: false,
```

Comment:
Looks like this `LOCAL` property isn't referenced anywhere in the codebase. Optional whether we want to remove it as part of this effort.